### ISSUE TITLE
file tests fail checks on setgid on directories

### ISSUE THREAD

**Issue by shalomb** [aelsabbahy/goss#65]
`goss` does not seem to be able to check if the `setuid`, `setgid` or `sticky` bits are set (for a directory in my case).

```
# cat svc-foo.yaml 
---
file:
  /usr/local/svc-foo:
    exists:   true
    filetype: directory
    mode:     "2775"
```

```
# chmod 2775 /usr/local/svc-foo/ -v
mode of ‘/usr/local/svc-foo/’ retained as 2775 (rwxrwsr-x)
```

The second test fails here when you don't expect it to.

```
# goss --gossfile svc-foo.yaml validate -f tap
1..3
ok 1 - File: /usr/local/svc-foo: exists: matches expectation: [true]
not ok 2 - File: /usr/local/svc-foo: mode: doesn't match, expect: ["2775"] found: ["0775"]
ok 3 - File: /usr/local/svc-foo: filetype: matches expectation: ["directory"]
```

**Comment by shalomb** [comment]
It seems to ignore the special bits entirely ..

```
# chmod -v 7775 /usr/local/svc-foo/                                         
mode of ‘/usr/local/svc-foo/’ changed from 2775 (rwxrwsr-x) to 7775 (rwsrwsr-t)
```

```
# goss --gossfile hub-web.yaml validate -f tap    
1..3
ok 1 - File: /usr/local/svc-foo: exists: matches expectation: [true]
not ok 2 - File: /usr/local/svc-foo: mode: doesn't match, expect: ["7775"] found: ["0775"]
ok 3 - File: /usr/local/svc-foo: filetype: matches expectation: ["directory"]
````

**Comment by aelsabbahy** [comment]
Fixed in [v0.1.4](https://github.com/aelsabbahy/goss/releases/tag/v0.1.4)
